src/components/LoadingButton/index.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+import {
+  Button,
+  InlineLoading
+} from '@carbon/react';
+
+import './styles.css';
+
+// The clickFn must provide a return to informing if the action
+// was successful or not, so the loading component can be properly
+// adjusted
+interface ButtonProps {
+  id: String,
+  clickFn: Function,
```

Comment:
This parameter needs to be as generic as possible, since the component may be used for different purposes, so I think leaving it a generic function works better.